src/login/loginctl.c

```diff
@@ -482,7 +482,6 @@ static int print_session_status_info(sd_bus *bus, const char *path, bool *new_li
                 { "Leader",              "u",    NULL,                     offsetof(SessionStatusInfo, leader)              },
                 { "Remote",              "b",    NULL,                     offsetof(SessionStatusInfo, remote)              },
                 { "Timestamp",           "t",    NULL,                     offsetof(SessionStatusInfo, timestamp.realtime)  },
```

Comment:
Then, how about to make `SessionStatusInfo.timestamp` `usec_t` instead of dual timestamp?

Reply:
The monotonic timestamp was actually used in `show_journal_by_unit`... Sorry for the noise.